bitnami/jaeger/templates/query/networkpolicy.yaml

```diff
@@ -33,24 +33,6 @@ spec:
           protocol: UDP
         - port: 53
           protocol: TCP
-    # Allow outbound connections to other collector pods
-    - ports:
-        - port: {{ .Values.collector.containerPorts.binary }}
-        - port: {{ .Values.collector.containerPorts.zipkin }}
-        - port: {{ .Values.collector.containerPorts.grpc }}
-        - port: {{ .Values.collector.containerPorts.admin }}
-      to:
-        - podSelector:
-            matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 14 }}
-              app.kubernetes.io/component: collector
-    # Allow outbound connections to other query pods
-    - ports:
-        - port: {{ .Values.query.containerPorts.admin }}
-        - port: {{ .Values.query.containerPorts.ui }}
-      to:
-        - podSelector:
-            matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 14 }}
-              app.kubernetes.io/component: query
```

Comment:
Why isn't this necessary any longer?

Reply:
I analized the application ports (https://www.jaegertracing.io/docs/2.8/architecture/apis/) and I think we made a wrong analysis of the Network policy ports to begin with.

In the documentation you can see 'Write API' and 'Read API' which are both inbound. 
Jaeger does not need to open any outbound connections to other jaeger collector or query nodes, as all the information is shared through the database.